yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/TimeValue.java

```diff
@@ -0,0 +1,55 @@
+package com.yoti.api.client.spi.remote;
+
+import static java.util.Calendar.HOUR_OF_DAY;
+import static java.util.Calendar.MILLISECOND;
+import static java.util.Calendar.MINUTE;
+import static java.util.Calendar.SECOND;
+
+import java.util.Calendar;
+
+import com.yoti.api.client.Time;
+
+public class TimeValue implements Time {
+
+    private final int hour;
+    private final int minute;
+    private final int second;
+    private final int millisecond;
+
+    private TimeValue(int hour, int minute, int second, int millisecond) {
+        this.hour = hour;
+        this.minute = minute;
+        this.second = second;
+        this.millisecond = millisecond;
+    }
```

Comment:
same here, you might want to introduce microseconds.

Reply:
Ok, have switched from milliseconds to microseconds